demos/segmentation_demo/python/segmentation_demo.py

```diff
@@ -96,12 +96,25 @@ def overlay_masks(self, frame, objects, output_transform):
         # Visualizing result data over source image
         return output_transform.resize(np.floor_divide(frame, 2) + np.floor_divide(self.apply_color_map(objects), 2))
 
+    def only_masks(self, objects, output_transform):
+        # Visualizing masks only
+        return output_transform.resize(self.apply_color_map(objects))
+
 
 class SaliencyMapVisualizer:
-    def overlay_masks(self, frame, objects, output_transform):
-        saliency_map = (objects * 255).astype(np.uint8)
+    def apply_color_map(self, input):
+        saliency_map = (input * 255.0).astype(np.uint8)
         saliency_map = cv2.merge([saliency_map, saliency_map, saliency_map])
-        return output_transform.resize(np.floor_divide(frame, 2) + np.floor_divide(saliency_map, 2))
+        return saliency_map
+
+    def overlay_masks(self, frame, objects, output_transform):
+        # Visualizing result data over source image
+        return output_transform.resize(np.floor_divide(frame, 2) + np.floor_divide(self.apply_color_map(objects), 2))
+
+    def only_masks(self, objects, output_transform):
```

Comment:
In C++ you don't introduce these funcs. Do the same for both.

If you keep the funcs, implementation for `SaliencyMapVisualizer` and `SegmentationVisualizer` are the same. Make `overlay_masks()` and `only_masks()` stand alone funcs. Or even merge them if you want to.